pwa/src/templates/pluginsDetailTemplate/PluginsDetailTemplate.module.css

```diff
@@ -153,3 +153,65 @@
 .showmoreVersionsButton {
   text-align: center;
 }
+
+/* readme styling */
+.readme {
+  padding-inline-end: var(--gateway-ui-size-md);
+  padding-block-start: var(--gateway-ui-size-md);
+  padding-block-end: var(--gateway-ui-size-md);
+}
```

Comment:
I would give the readme a smaller font size than the other text on the page.
Right now everyting looks really messy because there is a lot of text and with a smaller font size it looks les messy